src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectState.cs

```diff
@@ -440,6 +440,13 @@ public List<string> GetImportDocumentTargetPaths(HostDocument hostDocument)
             foreach (var importItem in importItems)
             {
                 var itemTargetPath = importItem.FilePath.Replace('/', '\\').TrimStart('\\');
+
+                if (FilePathComparer.Instance.Equals(itemTargetPath, hostDocument.TargetPath))
+                {
+                    // If an import change it can't impact itself. The above filepath normalization occasionally results in something like _Imports.razor mapping to /_Imports.razor because the leading / is trimmed.
```

Comment:
Could you elaborate this comment with an example? Even if this is a workaround, that will help us clean this up in the future.

nit: In the future, split comments like these in to two lines.